src/AzureDevOpsPolicyConfigurator/Logic/PolicyExecuterBase.cs

```diff
@@ -229,7 +228,7 @@ public async Task Execute(ExecuterSettings arguments)
 
                 if (!hasMatch)
                 {
-                    resultList.Add(() => this.CreatePolicy(policyClient, types, projectId, repository, currentPolicy, policy));
+                    resultList.Add(Task.Run(() => this.CreatePolicy(policyClient, types, projectId, repository, currentPolicy, policy)));
```

Comment:
Why calling `Task.Run` here?

Reply:
Did not find a better solution to await void methods:
https://stackoverflow.com/questions/13636648/wait-for-a-void-async-method

Reply:
Found a better way